src/pocketmine/block/Stone.php

```diff
@@ -77,12 +77,18 @@ public function getBreakTime(Item $item){
 
 	public function getDrops(Item $item){
 		if($item->isPickaxe() >= Tool::TIER_WOODEN){
-			return [
-				[Item::COBBLESTONE, 0, 1],
```

Comment:
Personally I wouldn't put two array brackets like that - it's not obvious that the array is supposed to be a list.